### DISCUSSION TITLE
2011/07/04/scratch%E7%BC%96%E7%A8%8B%E8%AF%AD%E8%A8%80

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#336] [Announcements]
# 2011/07/04/scratch%E7%BC%96%E7%A8%8B%E8%AF%AD%E8%A8%80

受八戒2师兄的提示，玩了一下Scratch这个软件。我前面介绍的Modit软件正是模仿了Scratch的编程方法和界面风格，所以两者在编程习惯上完全一致。但这两种软件是应用在完全不同的领域的。

https://blog.qizhen.xyz/2011/07/04/scratch%E7%BC%96%E7%A8%8B%E8%AF%AD%E8%A8%80

**Comment by ruanqizhen** [comment]
之前博客下的留言：

foolooo
2011年07月5日 15:34 编辑
请问您是怎样关联eordpress至微博的？

回复
ruanqizhen
2011年07月5日 17:13 编辑
你说新浪微博吗？它本身就提供了一项设置：“与博客关联”。

回复
foolooo
2011年07月7日 23:54 编辑
我也这么设置的，但是更新之后没有提示……奇怪了。

回复
ruanqizhen
2011年07月11日 10:58 编辑
我设置的时候似乎也有问题。不过我倒腾了几次，它自己就又工作了，我也不知道怎么回事。

回复
